src/scripts/oed.service

```diff
@@ -0,0 +1,28 @@
+# *
+# * This Source Code Form is subject to the terms of the Mozilla Public
+# * License, v. 2.0. If a copy of the MPL was not distributed with this
+# * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# *
```

Comment:
Files that users are required to edit during setup should not have the MPL header.